content/php/concepts/math-functions/terms/pi/pi.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'pi()'
+Description: 'Returns the value of PI.'
+Subjects:
+  - 'Web Development'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+  - 'Numbers'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`pi()`** function returns the value of PI.
+
+## Syntax
+
+```pseudo
+pi();
+```
+
+The `pi()` function has no parameters. It returns the value of PI as float. `M_PI`, being a constant, also returns the same value.
+
+## Example
+
```

Comment:
We need an introductory sentence for the example, even if it's just "The following example prints the value of pi to the console."

Reply:
noted on this

will edit accordingly